### PR TITLE
Anti-Capitalism: Add option to hide sponsored posts

### PR DESCRIPTION
**Pull request by ThePsionic**
This option honestly makes more sense to be in Anti-Capitalism.
I also removed the dim option on Tweaks, because that's pretty much useless at this point.
If this already handled somewhere else, let me know. I had a feeling this was already
part of XKit somewhere but I couldn't find it so I went ahead and wrote this up.

### CODE REVIEW

File: Extensions/anti_capitalism.js [hunk 1]
```diff
@@ -28,16 +33,20 @@ XKit.extensions.anti_capitalism = new Object({
...
-		if (XKit.extensions.anti_capitalism.preferences.vendor_buttons.value === true) {
+
+		if (XKit.extensions.anti_capitalism.preferences.vendor_buttons.value) {
 	       XKit.tools.add_css(" .post .vendor_button {display: none;}", "anti_capitalism_vendor_buttons");
 	    }
-	    
-	    if (XKit.extensions.anti_capitalism.preferences.sponsored_ads.value === true) {
+
+		if (XKit.extensions.anti_capitalism.preferences.sponsored_posts.value) {
+			XKit.tools.add_css(" .post.sponsored_post {display:none}", "anti_capitalism_sponsored_posts");
```
Comment (dlmarquis) [inline Extensions/anti_capitalism.js]: I can't get tumblr to show me sponsored posts anymore, but I know blocking the .sponsored class works as well.

**Comment by dlmarquis** [comment]
Do you think you could bring over the Hide Asktime option in Tweaks while you're at it? That should be moved here since Tumblr's using it for ads too now.

File: Extensions/anti_capitalism.js (hunk 1 above)
Comment (ThePsionic) [inline Extensions/anti_capitalism.js]: I copied the code over from Tweaks, and I am 100% sure that this works. Whichever way we choose to block it, it works as expected.

**Comment by dlmarquis** [comment]
Cool. LGTM.
@homu r+

File: Extensions/anti_capitalism.js [hunk 2]
```diff
@@ -19,25 +19,43 @@ XKit.extensions.anti_capitalism = new Object({
...
+
+		if (XKit.extensions.anti_capitalism.preferences.asktime.value) {
+			XKit.tools.add_css(" .notification.single_notification.alt.takeover-container { display: none; } ", "anti_capitalism_asktime");
+		}
 	},
-	
+
 
 	destroy: function() {
 		this.running = false;
```
Comment (Wolvan) [inline Extensions/anti_capitalism.js]: please destroy your CSS changes properly

**Pull request merged**